dogen/plugins/cct.py

```diff
@@ -53,9 +52,15 @@ def prepare(self, cfg):
         try:
             cmd = ['cct', '--modules-dir', cfg_file_dir, '-v', '--fetch-only', cfg_file]
             self.log.info("Executing %s" % " ".join(cmd))
-            out = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+
+            # setup cct to same logging level as dogen
+            cct_logger = logging.getLogger("cct")
```

Comment:
like this: 
https://github.com/containers-tools/cct/blob/master/cct/module.py#L30
logger works this way.